e2e_tests/tests/per_provider/normal_tests/key_agreement.rs

```diff
@@ -84,6 +87,13 @@ fn raw_key_agreement_secpr1() {
         return;
     }
 
+    // Section needed for crytpoauthlib as first ECC slot has unusual access behavior
+    // and user need to be aware of that.
```

Comment:
Could you give more details (not necessarily in the code - a comment here will do) about what that unusual access behavior is? Is this something documented for CAL?

Reply:
I have expanded this comment section a bit as I have learned that it depends on chip's configuration - which can be changed - and our current local testing configuration requires this code section. Parsec's universality is not helping in this case because we can't pass as parameter slot number or what type of ECC slot user would like to select.

I will try to figure out if we can implement some function to handle this case.